Makefile

```diff
@@ -87,14 +87,14 @@ endif
 endif
 
 _force_build:
-	DOCKER_BUILDKIT=1 docker build \
+	DOCKER_BUILDKIT=1 podman build \
```

Comment:
should this be changed to podman?

Reply:
This is a relic from testing on amd, will remove